examples/07_advanced/plot_haxby_mass_univariate.py

```diff
@@ -34,134 +34,166 @@
+        session_face_mask
+    ][0]
 
 ##############################################################################
 # Perform massively univariate analysis with permuted OLS
 #
 # We use a two-sided t-test to compute p-values, but we keep trace of the
 # effect sign to add it back at the end and thus observe the signed effect
 from nilearn.mass_univariate import permuted_ols
+
 neg_log_pvals, t_scores_original_data, _ = permuted_ols(
-    grouped_conditions_encoded, grouped_fmri_masked,
+    grouped_conditions_encoded,
+    grouped_fmri_masked,
     # + intercept as a covariate by default
-    n_perm=10000, two_sided_test=True,
-    verbose=1, # display progress bar
-    n_jobs=1)  # can be changed to use more CPUs
+    n_perm=10000,
+    two_sided_test=True,
+    verbose=1,  # display progress bar
+    n_jobs=1,
+)  # can be changed to use more CPUs
 signed_neg_log_pvals = neg_log_pvals * np.sign(t_scores_original_data)
 signed_neg_log_pvals_unmasked = nifti_masker.inverse_transform(
-    signed_neg_log_pvals)
+    signed_neg_log_pvals
+)
 
 ##############################################################################
 # scikit-learn F-scores for comparison
 #
 # F-test does not allow to observe the effect sign (pure two-sided test)
 from sklearn.feature_selection import f_regression
+
 _, pvals_bonferroni = f_regression(
-    grouped_fmri_masked,
-    grouped_conditions_encoded)  # f_regression implicitly adds intercept
+    grouped_fmri_masked, grouped_conditions_encoded
+)  # f_regression implicitly adds intercept
 pvals_bonferroni *= fmri_masked.shape[1]
 pvals_bonferroni[np.isnan(pvals_bonferroni)] = 1
 pvals_bonferroni[pvals_bonferroni > 1] = 1
 neg_log_pvals_bonferroni = -np.log10(pvals_bonferroni)
 neg_log_pvals_bonferroni_unmasked = nifti_masker.inverse_transform(
-    neg_log_pvals_bonferroni)
+    neg_log_pvals_bonferroni
+)
 
 ##############################################################################
 # Visualization
 import matplotlib.pyplot as plt
-from nilearn.plotting import plot_stat_map, show
 
 # Use the fmri mean image as a surrogate of anatomical data
-from nilearn import image
 from nilearn.image import get_data
+from nilearn.plotting import plot_stat_map, show
 
 mean_fmri_img = image.mean_img(func_filename)
 
 threshold = -np.log10(0.1)  # 10% corrected
 
-vmax = min(signed_neg_log_pvals.max(),
-           neg_log_pvals_bonferroni.max())
+vmax = min(signed_neg_log_pvals.max(), neg_log_pvals_bonferroni.max())
 
 # Plot thresholded p-values map corresponding to F-scores
-display = plot_stat_map(neg_log_pvals_bonferroni_unmasked, mean_fmri_img,
-                        threshold=threshold, cmap=plt.cm.RdBu_r,
-                        display_mode='z', cut_coords=[-1, ],
-                        vmax=vmax)
+display = plot_stat_map(
+    neg_log_pvals_bonferroni_unmasked,
+    mean_fmri_img,
+    threshold=threshold,
+    cmap=plt.cm.RdBu_r,
+    display_mode="z",
+    cut_coords=[
+        -1,
```

Comment:
:-1:

Reply:
This one can be fixed by removing the comma

Reply:
indeed.
now fixed